src/components/LoadingIndicator/LoadingIndicator.module.css

```diff
@@ -12,6 +12,9 @@
 }
 
 .loading-indicator__icon {
+  height: 3rem;
+  width: 3rem;
+  fill: var(--eds-theme-color-icon-brand-primary);
```

Comment:
`color` and `size` props should be updated in `LoadingIndicator.tsx` instead, when calling `<Icon />`